Form/Type/BasePickerType.php

```diff
@@ -145,6 +145,8 @@ protected function getCommonDefaults()
             'dp_collapse' => true,
             'dp_calendar_weeks' => false,
             'dp_view_mode' => 'days',
+            'dp_min_view_mode' => 'days',
+            'dp_max_view_mode' => 'centuries',
```

Comment:
There is no `maxViewMode` option in eonasdan/bootstrap-datetimepicker 3.1.3. You can find it in other forks like uxsolutions/bootstrap-datepicker but not this one.